client/state/selectors/is-eligible-for-wpcom-monthly-plan.js

```diff
@@ -21,11 +21,7 @@ export default createSelector(
 			return true;
 		}
 
-		return (
-			( isAtomicSite( state, siteId ) && currentPlanSlug === 'jetpack_free' ) ||
-			isWpComMonthlyPlan( currentPlanSlug ) ||
-			isWpComFreePlan( currentPlanSlug )
-		);
+		return isWpComMonthlyPlan( currentPlanSlug ) || isWpComFreePlan( currentPlanSlug );
```

Comment:
I think this needs D75518-code to land first, so let's make sure this PR doesn't get deployed before 🙂